image_tools/src/showimage.cpp

```diff
@@ -80,73 +86,75 @@ void show_image(
   }
 }
 
-int main(int argc, char * argv[])
+/// Default constructor for no command line args
+/**
+ */
```

Comment:
Remove empty doc block (or populate). 

Also, I think all of the doc blocks can be moved to the header files.